runtime/autoload/health/provider.vim

```diff
@@ -496,7 +496,7 @@ function! s:check_node() abort
     return
   endif
 
-  if !executable('node') || !executable('npm') || !executable('yarn')
+  if !executable('node') || !executable('npm')
```

Comment:
So if yarn is available but not npm, it still fails?

Reply:
(I updated my commit msg, but not PR msg)

* yes, it's highly unlikely you have yarn installed but not npm
* we also only try to call `npm`: https://github.com/neovim/neovim/blob/master/runtime/autoload/health/provider.vim#L516